provisioning_templates/snippet/kickstart_networking_setup.erb

```diff
@@ -8,35 +8,43 @@ description: this will configure your host networking, it configures your primar
 %>
 <% subnet = @host.subnet -%>
 <% subnet6 = @host.subnet6 -%>
-<% dhcp = subnet.dhcp_boot_mode? -%>
+
+rm -v -f /etc/sysconfig/network-scripts/ifcfg-eth*
```

Comment:
This is copied from a copy of a script from some time ago; it does not assume a clean system and runs on pre-installed systems to change their network.